app/angular/src/builders/start-storybook/index.spec.ts

```diff
@@ -119,7 +120,6 @@ describe('Start Storybook Builder', () => {
       sslCa: undefined,
       sslCert: undefined,
       sslKey: undefined,
-      staticDir: [],
```

Comment:
is this a breaking change? we deprecated the `staticDir` CLI option in 6.4 but won't remove it until 7.0

Reply:
I don't think many people use the builder, let alone this option :thinking: 
as this is not the default behaviour nor really a requirement for SB angular
IMHO the builders are still an experiment. But if it works well, we can put them as default (Docs + init cmd) 

and when I was doing my tests, I couldn't find any other way than to delete it to avoid the error :/ 
```
ERR! Error: Conflict when trying to read staticDirs:
ERR! * Storybook's configuration option: 'staticDirs'
ERR! * Storybook's CLI flag: '--staticDir' or '-s'
```

but if necessary and if there is another way to do it we can revert this commit ;)